### ISSUE TITLE
Invalid plugin phpArray

### ISSUE THREAD

**Issue by xorock** [zendframework/zend-i18n#55]
I'm using delegator factory as shown in docs for zend-i18n-resources (https://github.com/zendframework/zend-i18n-resources/blob/master/doc/book/usage.md). I'm not sure if it's a bug inside documentation or missing alias for Service Manager.
https://github.com/zendframework/zend-i18n/blob/master/src/Translator/LoaderPluginManager.php#L62
Here We can see definition only for lowercased 'phparray' key, so 'phpArray' won't work.

Another thing is delegator config:

``` php
return [
    'service_manager' => [
        'delegators' => [
            'MvcTranslator' => [
                'TranslatorDelegator',
            ],
        ],
    ],
];
```

`MvcTranslator` key will not trigger TranslatorDelegator (I'm using this to translate validator messages inside form generated with Zend\Form\View\Helper\Form). Instead, I had to create this as follows:

``` php
use Zend\Mvc\I18n\Translator;
...
'service_manager' => [
        'delegators' => [
            Translator::class => [
                TranslatorDelegator::class,
            ],
        ],
    ],
```

Shouldn't locale be normalized and maybe even merged, for example

``` php
'translator' => [
        'locale' => 'pt_BR',
    ],
```

First, class should look for `pt_BR` translation, then `pt` and if both exist, merge them.

**Comment by samsonasik** [comment]
there is PR for it https://github.com/zendframework/zend-i18n/pull/56

**Comment by froschdesign** [comment]
Fixed with #56
